Objects/Objects/Geometry/Orientation.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Speckle.Core.Models;
+
+namespace Objects.Geometry
+{
+  public class Orientation : Base
```

Comment:
New `Orientation` class is probably not necessary, could use `Plane` or 'Transform` instead depending on the use cases